code/modules/mob/living/carbon/human/human.dm

```diff
@@ -313,6 +313,9 @@ var/list/rank_prefix = list(\
 //Now checks siemens_coefficient of the affected area by default
 /mob/living/carbon/human/electrocute_act(shock_damage, obj/source, siemens_coeff = 1.0, def_zone = null)
 	if(status_flags & GODMODE)	return 0	//godmode
+	if(shock_damage > 50)
```

Comment:
I would check shock damage multiplied by the siemens coeff, else you could just touch a live wire with an insulated glove and this breakdown is essentially useless.

Do it after the get_siemens_coefficient_organ check

Reply:
Fair point.

Reply:
And if that's will be the real damage, that's almost enough to burn your limb away. Value of 50 is clearly too high, lower it to 10 so user will be able to just mildly shock himself to conclude breakdown.

Reply:
You kinda forgot to remove this after adding the new method of high voltage conclusion